### PR TITLE
Richardjm/back plate

### PR DESCRIPTION
**Pull request by richardjm**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

Back blanking plate

#### How was it tested? 

In use on my Voron 2.4 since I made it

#### Any background context you want to provide?

I wasn't intending to PR this as it's a pretty simple mod but a couple of people asked about it lately so I neated it up and made proper instructions and many depth variants (along with parameterised CAD) to suit.

#### Screenshots (if appropriate)

See readme

#### Further notes

It's a big ol' lump of plastic so prone to warping. Was one of the very first prints on my Voron to replace the tape at the back as I never made the exhaust filter housing and just went straight to nevermore.

### CODE REVIEW

File: printer_mods/richardjm/back-plate/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Back plate blanking plate
+
+If you choose not to use the exhaust - perhaps you're planning on fitting a nevermore instead. This is a blanking plate with integrated bowden tube route for that goes in place of the exhaust filter housing.
```
Comment (KiloQubit) [inline printer_mods/richardjm/back-plate/README.md:3]: I think the word `for` in this sentence wasn't intended to be there

> This is a blanking plate with integrated bowden tube route for that goes in place of the exhaust filter housing

File: printer_mods/richardjm/back-plate/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+It makes use of the existing grill with one strut removed (use the stl here or cut out a strut of a grill you already have printed) as I liked the look of it.
+
+![The CAD - outside](Images/CAD1.png)
+
+![The CAD - inside](Images/CAD2.png)
+
+## Insructions
```
Comment (KiloQubit) [inline printer_mods/richardjm/back-plate/README.md:10]: Replace `Insructions` with `Instructions`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Shouldn't there also be a 4mm version as well, since that is the stock configuration?  3mm ABS panel + 1mm foam:
Screen Shot 2021-10-08 at 08 20 57
Screen Shot 2021-10-08 at 08 22 32

Please also see README changes below.

File: printer_mods/richardjm/back-plate/README.md (hunk 2 above)
Comment (richardjm) [reply printer_mods/richardjm/back-plate/README.md]: Fixed

**Pull request merged**